src/main/scala/EmrSparkPlugin.scala

```diff
@@ -261,37 +293,41 @@ object EmrSparkPlugin extends AutoPlugin {
     }
   }
 
-  def submitJob(
-    settings: Settings,
-    mainClass: String,
-    args: Seq[String],
-    jar: File
-  )(implicit log: Logger) = {
-    log.info("Uploading the jar.")
-
+  def uploadJarToS3(s3Location: String, jarFile: String)(implicit log: Logger) = {
```

Comment:
Could we use `jarFile: File` here to avoid someone who send a strange String in?